### PR TITLE
feat: add short summary in update and create pool

### PR DESCRIPTION
**Pull request by ChristanDaniel**
## Why
 Introduce a new input 'short_summary' in the pool creation and update processes.

## Changes
- Implement a new component called 'TextareaWithValueCounter
- Integrate this component into pool creation and update processes
- Update the query to accommodate the new 'short_summary' field

### CODE REVIEW

File: src/components/TextareaWithValueCounter/index.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+interface ITextareaWithValueCounterProps {
+  form?: string
+  name: string
+  type: string
+  value: string
+  required?: boolean
+  placeholder: string
+  minLength: number
+  maxLength: number
+  lable: string
```
Comment (Bmullerc) [inline src/components/TextareaWithValueCounter/index.tsx:12]: typo?

File: src/components/TextareaWithValueCounter/index.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  placeholder,
+  minLength,
+  maxLength,
+  lable,
+  readonly = false,
+  onChange
+}: ITextareaWithValueCounterProps) => {
+  return (
+    <S.TextareaWithValueCounter>
+      <S.Label htmlFor={name}>{lable}</S.Label>
```
Comment (Bmullerc) [inline src/components/TextareaWithValueCounter/index.tsx:31]: same as above, typo?

File: src/templates/Manage/CreatePool/SetDetails/PoolDetails/index.tsx [hunk 3]
```diff
@@ -62,6 +66,18 @@ const PoolDetails = () => {
...
+
+        <TextareaWithValueCounter
+          form="poolCreationForm"
+          name="poolShortSummary"
+          type="text"
+          placeholder="Enter a Brief Description"
+          value={details.shortSummary ?? ''}
+          minLength={0}
+          maxLength={150}
+          lable="DESCRIPTION"
```
Comment (Bmullerc) [inline src/templates/Manage/CreatePool/SetDetails/PoolDetails/index.tsx:78]: same as above, typo?

File: src/templates/PoolManager/Details/Strategy/index.tsx [hunk 4]
```diff
@@ -97,6 +110,20 @@ const Strategy = () => {
...
+      <S.InputCountWrapper>
+        <TextareaWithValueCounter
+          name="shortSummary"
+          type="text"
+          placeholder="Enter a Brief Description"
+          required
+          value={shortSummary}
+          minLength={0}
+          maxLength={150}
+          lable="DESCRIPTION"
```
Comment (Bmullerc) [inline src/templates/PoolManager/Details/Strategy/index.tsx:122]: same as above, typo?

**Review by Bmullerc — APPROVED** [review]
LAGTM

**Review by nicholascostadev — APPROVED** [review]
LGTM

**Pull request merged**